### PR TITLE
Doc bug-fixes.

### PR DESCRIPTION
**Pull request by bkpratt84**
Updated the default value for props dark and light for theme props.

Updated ComponentParameters component to remove duplicate props.

Fixed spelling mistake. Themeing should be theming.

### CODE REVIEW

File: components/ComponentParameters.vue [hunk 1]
```diff
@@ -306,7 +305,7 @@
           [
             'light',
             'Boolean',
-            'False',
+            'True',
```
Comment (johnleider) [inline components/ComponentParameters.vue]: I don't understand this change

File: components/ComponentParameters.vue [hunk 2]
```diff
@@ -422,7 +421,7 @@
           [
             'dark',
             'Boolean',
-            'False',
+            'True',
```
Comment (johnleider) [inline components/ComponentParameters.vue]: or this one

File: components/ComponentParameters.vue (hunk 1 above)
Comment (bkpratt84) [reply components/ComponentParameters.vue]: https://vuetifyjs.com/components/sliders

Look at the above page. The props light and dark both show twice. One has the default value as true and one has it as false.

File: components/ComponentParameters.vue (hunk 2 above)
Comment (bkpratt84) [reply components/ComponentParameters.vue]: Same as above.

**Comment by johnleider** [comment]
Okay, that makes sense. Last question would be the 0.13 file changes. Were they just text changes as well?

**Comment by bkpratt84** [comment]
Yes, I just changed the spelling of Themeing to Theming.

**Pull request merged**